src/pages/singleListing.js

```diff
@@ -0,0 +1,79 @@
+import Link from "next/link";
+import NavBar from "@/components/Header/NavBar";
+import CustomizableAvatar from "@/components/common/customavatar/CustomizableAvatar";
+import CustomizableButton from "@/components/common/CustomizableButton";
+import FooterBar from "@/components/common/FooterBar";
+import styles from "../styles/singleListing.module.css";
+
+
+function SingleListing(className, text) {
```

Comment:
@Sheyitofunmi there is this file `src/pages/[community_id].js` that I believe is the SingleListing page. In this PR you are creating a new but unnecessary one. Please make your changes to the `src/pages/[community_id].js` file and delete this file, thanks!

Reply:
alright ma, will make changes

Reply:
@Sheyitofunmi Please delete this file: src/pages/singleListing.js